plugins/LilyPad/Config.h

```diff
@@ -23,9 +23,13 @@ extern u8 ps2e;
 enum PadType {
 	DisabledPad,
 	Dualshock2Pad,
-	GuitarPad
+	GuitarPad,
+	PopnPad
 };
 
+const int numPadTypes= 4;
+extern const wchar_t *padTypes[numPadTypes];
+
```

Comment:
Hmm, I'm not even sure why I commented that above. maybe I ignored the ``DisabledPad`` ? either way please ignore that comment. :)